tests/runtimes/local/runtime_test.py

```diff
@@ -280,3 +282,41 @@ def testScanInLocalRuntime_whenScanIdIsPassed_shouldUseTheScanIdAsUniverseLabelI
         assert session.query(models.Scan).count() == 1
         scan = session.query(models.Scan).first()
         assert scan.id != 42
+
+
+@pytest.mark.docker
+def testRuntime_WhenCantInitSwarm_shouldShowUserFriendlyMessage(
```

Comment:
`_shouldShowUserFriendlyMessage` ... I don't see a check for the message in the unit test.